server/app.js

```diff
@@ -36,6 +37,7 @@ var app = express();
 app.use(logger("dev"));
 app.use(express.json()); // to accept JSON data in the body
 app.use(json());
+app.use(cors({ credentials: true, origin: "http://localhost:3000" }));
```

Comment:
are these CORS required or is this leftover from trying to debug the token issue?

Reply:
I believe credentials is required, but origin isn't. I'll remove it now